app/views/layouts/_application_content.html.erb

```diff
@@ -2,20 +2,31 @@
   <div id="content" >
     <%= notifications %>
     <% unless @page_header.blank? %>
-      <div class="row form-group">
-        <h1 class="col-md-8"><%=h @page_header %></h1>
-      </div>
+        <div class="row form-group">
+          <h1 class="col-md-8"><%=h @page_header %></h1>
+        </div>
+    <% end %>
+    <% if @count_nil_hosts != nil && @count_nil_hosts > 0 %>
+        <%= alert :class => 'alert-warning', :header => '',
+                  :text => (_("There are #{@count_nil_hosts}") + ' ' +
+                      link_to(_("%{count} hosts with no %{taxonomy_single} assigned") %
+                                  {:count => @count_nil_hosts , :taxonomy_single => taxonomy_single },
+                              hosts_path(:search => "not has #{controller_name.singularize}"))).html_safe %>
+    <% end %>
+    <% if searchable? && !(@page_header == _("Audits") || @page_header == _("Overview") )  %>
```

Comment:
You can check `controller_name` here instead of `@page_header` to be independent of string translations and changes.